src/linkplay/utils.py

```diff
@@ -67,6 +75,50 @@ async def session_call_api_ok(
         raise LinkPlayRequestException(f"Didn't receive expected OK from {endpoint}")
 
 
+async def call_tcpuart(host: str, cmd: str) -> str | None:
```

Comment:
I am wondering if we should supply the socket here, coming from the endpoint class, rather than remaking it every time